lib/Wikia/tests/Util/SamplerProxyTest.php

```diff
@@ -284,6 +288,18 @@ function testExceptionFallbackFromShadow() {
 		$this->assertEquals( $originalTestResult, $result );
 	}
 
+	function getRandomInt() {
+		static $intArray;
+		static $index = 1;
+
+		if ( !isset( $intArray ) ) {
+			$intArray = range( 1, 100 );
```

Comment:
`100` here is closely connected with `callCount`. If we change `100` to `1000` we can again land into not perfect rand distribution problem. Can we make sure other devs know that by using a variable or putting a comment?